optimism.tokenlist.json

```diff
@@ -593,6 +593,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+    {
+      "chainId": 1,
+      "address": "0x15Ee120fD69BEc86C1d38502299af7366a41D1a6",
+      "name": "BitANT",
+      "symbol": "BitANT",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BitANT.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x5029C236320b8f15eF0a657054B84d90bfBEDED3",
```

Comment:
This implementation allows tokens to be burnt on L2 meaning tokens on the L1 Optimism standard bridge will remain locked eternally. Is that intended behaviour?

Reply:
Yup. 
Cuz BitANT was first deployed on Ethereum mainnet, 10 billion BitANT have been minted at genesis, and the cap of BitANT is fixed. When BitANT on Optimism goes back to Ethereum mainnet, it will be burned. Once the tx message reaches L1 through the relay, the L1 standard bridge will release the token and transfer it to the target address.

Reply:
I was not talking about the generic way a bridge works, I understand that. I was talking about the scenario where people burn bridged tokens on L2 thus making them "untransferable" back to L1 e.g.
- User deposits 100 tokens on the bridge
- User receives their 100 tokens on L2
- User burns 20 tokens on L2
- User withdraws remaining 80 tokens back to L1
- 20 tokens are left forever in the bridge being non-withdrawable

Reply:
Sorry, GT.
That's indeed intended behaviour. Cuz our platform charges exchange fees when users swap WBTC to BitBTC and vice versa. Exchange fee is mainly used to repurchase and burn BitANT.